backend/scripts/init_vault.py

```
@@ -0,0 +1,234 @@
+        parts = db_url.replace("postgresql://", "").split("/")
+        db_name = parts[-1] if len(parts) > 1 else "jeex_plan"
+
+        host_part = parts[0].split("@")[-1] if "@" in parts[0] else "postgres:5432"
+        host, port = host_part.split(":") if ":" in host_part else (host_part, "5432")
+
+        user_pass = parts[0].split("@")[0] if "@" in parts[0] else "jeex_user:jeex_password"
+        user, password = user_pass.split(":") if ":" in user_pass else (user_pass, "jeex_password")
+    else:
+        # Default values
+        user = "jeex_user"
+        password = "jeex_password"
```

Ruff (0.13.1)

Possible hardcoded password assigned to: "password"

(S105)